### ISSUE TITLE
KeyError: 'input_values'

### ISSUE THREAD

**Issue by ProfessorVenkman** [fyookball/electrum#661]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "site-packages\electron_cash-3.1.6-py3.5.egg\electroncash_gui\qt\main_window.py", line 2226, in do_process_from_file
  File "site-packages\electron_cash-3.1.6-py3.5.egg\electroncash_gui\qt\main_window.py", line 2206, in read_tx_from_file

KeyError: 'input_values'
```

Reporter
------------

This issue was reported by 2 user(s):

| Electrum Version  | Operating System  | Wallet Type  | Locale |
|---|---|---|---|
| 3.1.6  | Windows-10-10.0.16299 | standard | en_US |
| 3.1.6  | Windows-10-10.0.16299 | standard | en_US |

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by rt121212121** [comment]
@kyuupichan You merged in commit 922b133cb02982cd687a73195b0f598a17fb1152 which both added this extra dictionary entry, and broke older signed transactions.  Is there a way to change the tx loading so it knows what to do if this entry is not there?

**Comment by rt121212121** [comment]
Never mind.  I guess it's added for users of the exported files, not for any necessary purpose, given that's what the commit suggests.  I'll do a pull request based on that.
